modules/acquia_lift_publisher/src/EventSubscriber/Publish/PublishOnlyRendered.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+namespace Drupal\acquia_lift_publisher\EventSubscriber\Publish;
+
+use Acquia\ContentHubClient\CDF\CDFObject;
+use Drupal\acquia_contenthub\AcquiaContentHubEvents;
+use Drupal\acquia_contenthub\ContentHubCommonActions;
+use Drupal\acquia_contenthub\Event\PrunePublishCdfEntitiesEvent;
+use Drupal\acquia_contenthub_publisher\ContentHubPublisherEvents;
+use Drupal\acquia_contenthub_publisher\Event\ContentHubEntityEligibilityEvent;
+use Drupal\acquia_lift_publisher\Form\AcquiaLiftPublisherSettingsForm;
+use Drupal\Core\Config\ImmutableConfig;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+/**
+ * Responsible for filtering contents based on publishing configurations.
+ *
+ * @see \Drupal\acquia_lift_publisher\Form\AcquiaLiftPublisherSettingsForm::$pushSettingField
+ * @package Drupal\acquia_lift_publisher\PruneCdf
+ */
+class PublishOnlyRendered implements EventSubscriberInterface {
+
+  /**
+   * Content Hub common actions.
+   *
+   * @var \Drupal\acquia_contenthub\ContentHubCommonActions
+   */
+  protected $commonActions;
```

Comment:
Please change all `protected` functions and variables to `private`, unless they are intended for inheritance.

I understand Content Cloud projects use minimally `protected`. `acquia_lift` modules obeys the universal standards of [Information Hiding](https://en.wikipedia.org/wiki/Information_hiding).

Reply:
Noted.